<?xml version="1.0"?>
<launch>
  <param name="/use_sim_time" value="true"/>

  <include file="$(find scout_description)/launch/load_base_model.launch"/>

  <arg name="auto-close" default="false" doc="Startup mode"/>
  <node name="scout_webots_node" pkg="scout_webots_sim" type="scout_webots_node" required="$(arg auto-close)"  output="screen">
    <param name="use_sim_time" value="true" />
    <param name="clock" value="/clock" />
    <param name="odom_frame" type="string" value="odom" />
    <param name="base_frame" type="string" value="base_link" />
    <param name="simulated_robot" type="bool" value="true" />
    <param name="sim_control_rate" value="50" />
  </node>

</launch>
